driver-service/src/test/java/com/modsen/driverservice/AppTestUtil.java

```diff
@@ -10,9 +10,10 @@
 import java.util.ArrayList;
 import java.util.Collections;
 
-public class AppTestUtil {
+public final class AppTestUtil {
+
+    private AppTestUtil() {}
```

Comment:
Для всех `AppTestUtil`, раз уж подключен lombok.
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class AppTestUtil {
```